Implements the function to issue notification using ntfy (#31)
mwakaba2/jupyterlab-notifications: README.md

    // The minimum execution time to send out notification for a particular notebook cell (in seconds).
    "minimum_cell_execution_time": 60,

    // Notification Methods

>     // Methods how to notificate messages. Select from 'browser' or 'ntfy'

Reviewer: Could you update the description to this? I just to make sure the users understand that they can choose more than one.
```suggestion
    // Option to send a notification with the specified method(s). The available options are 'browser' and 'ntfy'.
```

![notification](https://user-images.githubusercontent.com/3497137/111881088-01db5200-897d-11eb-8faa-4701cabfcde4.gif)

> ## (Option) Notifications using `ntfy`

Reviewer: ```suggestion
## (Optional) Notifications using `ntfy`
```

You can recieve notifications via `ntfy`.

**ntfy 2.7.0 documentation** https://ntfy.readthedocs.io/en/latest/

> > ntfy brings notification to your shell. It can automatically provide desktop notifications when long running commands finish or it can send push notifications to your phone when a specific command finishes.

Reviewer: ```suggestion
> ntfy brings notification to your shell. It can automatically provide desktop notifications when long running code executions finish or it can send push notifications to your phone when a specific execution finishes.
```

> ### How to work

Reviewer: ```suggestion
### How to enable notifications via `ntfy`
```

Install `ntfy`.

```console
$ pip install ntfy
```

> Configure ntfy in accodance with [the ntfy official document](https://ntfy.readthedocs.io/en/latest/#configuring-ntfy)

Reviewer: ```suggestion
To configure ntfy, please check out [the ntfy official configuration docs](https://ntfy.readthedocs.io/en/latest/#configuring-ntfy)
```

```console
$ vim ~/.config/ntfy/ntfy.yml
```

```yaml
backends:
    - pushover
pushover:

>     user_key: hunter2

Reviewer: I'm not familiar with `ntfy`. Do users have to create their own user_key to enable `ntfy`? Or will they use the same `hunter2` user_key?

Author: @mwakaba2 
No, intergration with pushover is just only option, and `hunter2` implies `YOUR_PUSHOVER_USER_KEY`. The code on `How to enable notifications via ntfy` subsection are example codes.

It seems to be poorly-written.... so I'll also update sentenses on this subsection.

Reviewer: Sounds great! Thanks for the clarification and doc update!
